packages/vscode-zipfs/rollup.config.js

```diff
@@ -19,7 +19,7 @@ export default async () => ({
       jail: path.join(__dirname, `../../`),
       preferBuiltins: true,
     }),
-    esbuild({tsconfig: false, target: `node12`, format: `cjs`}),
```

Comment:
`format` isn't a valid option for the esbuild plugin, I'm not sure how I managed to introduce it in https://github.com/yarnpkg/berry/pull/3683.